community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tftpl

```diff
@@ -51,27 +52,15 @@
     ansible.builtin.command: git clone --branch {{ git_ref }} {{ git_url }} {{ install_dir }}
     when: lock_out.rc == 0
 
-  - name: chgrp on installation
+  - name: Set ownership and permissions
     ansible.builtin.file:
       path: "{{ install_dir }}"
-      group: "{{ chgrp_group }}"
+      mode: "{{ chmod_mode | default(omit, true)  }}"
+      owner: "{{ chown_owner | default(omit, true) }}"
+      group: "{{ chgrp_group | default(omit, true)  }}"
       recurse: true
-    when: chgrp_group != "" and lock_out.rc == 0
-
-  - name: chown on installation
-    ansible.builtin.file:
-      path: "{{ install_dir }}"
-      owner: "{{ chown_owner }}"
-      recurse: true
-    when: chown_owner != "" and lock_out.rc == 0
-
-  - name: chmod on installation
-    ansible.builtin.file:
-      path: "{{ install_dir }}"
-      mode: "{{ chmod_mode }}"
-      recurse: true
-    when: chmod_mode != "" and lock_out.rc == 0
+    when:  lock_out.rc == 0
```

Comment:
there's a bit of extra whitespace here that I think was in my comment. Might as well remove it.

Reply:
Done.